honeybee_display/model.py

```diff
@@ -490,7 +490,7 @@ def _read_sensor_grid_result(result_folder):
         # find the result file and append the results
         result_file = None
         for f in os.listdir(result_folder):
-            if f.startswith(grid_id):
+            if f.startswith(grid_id) and not grid_id.endswith('.json'):
```

Comment:
hey @chriswmackey - it should probably be f.endswith.

```suggestion
            if f.startswith(grid_id) and not f.endswith('.json'):
```

Reply:
Thank you. Fixed!